test/collections/Vector2DTest.ooc

```diff
@@ -10,113 +10,38 @@ use unit
 use collections
 
 Vector2DTest: class extends Fixture {
-	vector2D: Vector2D<Int>
```

Comment:
Side effects between test cases made it hard to see what failed when adapting to x,y.